### ISSUE TITLE
Redirection after login from a page with special characters truncates page name

### ISSUE THREAD

**Issue by phbaudin** [multidimension-al/phpbbauth#6]
If a user logs in from a page on the wiki with special characters in the title, phpBB will redirect him to the original link, only truncated from the first special character.

A simple `urlencode` in Phpbbauth/PhpbbAuthHooks.php seems to solve the issue:

* Line 28

Replaced:
```
			$personal_urls['login']['href'] = $wgPhpBBAuthForumDirectory . 'ucp.php?mode=login&redirect=' . $_SERVER[ 'REQUEST_URI' ];
```
with:
```
			$personal_urls['login']['href'] = $wgPhpBBAuthForumDirectory . 'ucp.php?mode=login&redirect=' . urlencode( $_SERVER[ 'REQUEST_URI' ] );
```

* Line 32

Replaced:
```
			$personal_urls['anonlogin']['href'] = $wgPhpBBAuthForumDirectory . 'ucp.php?mode=login&redirect=' . $_SERVER[ 'REQUEST_URI' ];
```
with:
```
			$personal_urls['anonlogin']['href'] = $wgPhpBBAuthForumDirectory . 'ucp.php?mode=login&redirect=' . urlencode( $_SERVER[ 'REQUEST_URI' ] );
```

**Comment by ajquick** [comment]
Thank you. This seems to be an obvious and straight forward fix. Now if only there were any easy fix to override the phpBB code that prevents redirecting to a folder outside of phpBB!

**Comment by ajquick** [comment]
Fixed in version v0.0.6, including the addition of the $wgServer variable for absolute pathing. Useful for installations where the forum and wiki are on seperate domains or subdomains.

https://github.com/multidimension-al/phpbbauth/releases/tag/0.0.6
